modules/tests/test/dataobject/operations.go

```diff
@@ -56,7 +56,7 @@ func HasDataVolumeFailedToImport(dataVolume *cdiv1beta1.DataVolume) bool {
 	conditions := getConditionMapDv(dataVolume)
 	return dataVolume.Status.Phase == cdiv1beta1.ImportInProgress &&
 		dataVolume.Status.RestartCount > constants.UnusualRestartCountThreshold &&
-		conditions[cdiv1beta1.DataVolumeBound].Status == v1.ConditionTrue &&
+		conditions[cdiv1beta1.DataVolumeBound].Status == v1.ConditionFalse &&
```

Comment:
Can the check be dropped instead? Do the other conditions suffice to assert the import failed?

Reply:
Yeah, probably the rest of the asserts are enough to check if the DV has failed. WDYT @ksimon1?

Reply:
fine by me

Reply:
Done!